README.md

```diff
@@ -10,19 +10,37 @@
 
 ## Description
 
-This module provides helper tasks that are used by the `apply()` functionality in Bolt. These tasks are not intended to be run on their own.
+This module provides helper tasks that are used by the `apply()` functionality in [Bolt](https://github.com/puppetlabs/bolt). These tasks are not intended to be run on their own, but as part of a [bolt plan](https://puppet.com/docs/bolt/latest/writing_plans.html) that uses the [apply keyword](https://puppet.com/docs/bolt/latest/applying_manifest_blocks.html).
 
 This module allows Bolt plans using `apply()` to be run on a Puppet Enterprise (PE) installation.
 
 ## Setup
 
-Install the `apply_helpers` module in PE. Using PE RBAC, grant the ability to run the `apply_helpers::custom_facts` and `apply_helpers::apply_catalog` tasks to those you intend to use this module.
-
-To use Bolt's `apply_prep` function, also install https://github.com/puppetlabs/puppetlabs-puppet_agent's `master` branch.
+1. First you'll need to install the [puppetlabs-puppet_agent](https://github.com/puppetlabs-puppet_agent) module from the master branch on github. The easiest way to do this is to add
```

Comment:
I'd lean towards sticking with the Puppetfile way of doing things. The git workflow works fine.

Reply:
I'm assuming that means keep it as is?